LootLockerServerSDK/Source/LootLockerServerSDK/Public/ServerAPI/LootLockerServerAssetRequest.h

```diff
@@ -604,6 +604,59 @@ struct FLootLockerServerSimpleAsset
     TArray<FLootLockerServerMetadataEntry> metadata;
 };
 
+/**
+ * Represents a context for organizing assets and inventory
+ */
+USTRUCT(BlueprintType)
+struct FLootLockerServerContext
+{
+    GENERATED_BODY()
+    /**
+     * The unique identifier of the context
+     */
+    UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+    int Id = 0;
+    /**
+     * The UUID of the context
+     */
+    UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+    FString Uuid = "";
+    /**
+     * The name of the context
+     */
+    UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+    FString Name = "";
+    /**
+     * The human-friendly name of the context
+     */
+    UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+    FString Friendly_name = "";
+    /**
+     * Whether assets in this context can be detached
+     */
+    UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+    bool Detachable = false;
+    /**
+     * Whether this context is visible to players
+     */
+    UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+    bool User_facing = false;
+    /**
+     * Optional: The ID of a parent asset if this context is nested
+     * 
+     * This variable is optional meaning it may or may not exist, which is why it's a string. To get the value from it you should first check if it is valid (for example using .IsNumeric() and then get the value from it (for example using FCString::Atoi)
+     */
+    UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+    FString Dependent_asset_id = "";
+    /**
+     * Optional: Maximum number of assets that can be equipped in this context
+     * 
+     * This variable is optional meaning it may or may not exist, which is why it's a string. To get the value from it you should first check if it is valid (for example using .IsNumeric() and then get the value from it (for example using FCString::Atoi)
+     */
+    UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+    FString Max_equip = "";
+};
+
```

Comment:
This is hallucination. The only fields in the context are:
`id` - Integer ID of the context
`uuid` - Context UUID
`name` - Context Name